src/main/kotlin/step2/Calculator.kt

```diff
@@ -0,0 +1,48 @@
+package step2
+
+import java.lang.ArithmeticException
+
+fun main() {
+    println(Calculator(readLine().toString()).caculate())
+}
+
+class Calculator(private var inputText: String) {
```

Comment:
```suggestion
class Calculator(private val inputText: String) {
```

inputText 의 값을 직접적으로 수정하는 로직이 없는 듯 한데,
val 이 아니라 var 로 선언하신 이유가 있으시나요?